### PR TITLE
add disclaimer

### PR DESCRIPTION
**Pull request by pjfanning**
see #34

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: DISCLAIMER [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Apache Pekko (incubating) is an effort undergoing incubation at the Apache
```
Comment (Seetaramayya) [inline DISCLAIMER:1]: Is it not `Apache Pekko Connector Kafka (incubating)`?

Comment (pjfanning) [reply DISCLAIMER:1]: the project name is Apache Pekko - that's what the disclaimer is about

Comment (pjfanning) [reply DISCLAIMER:1]: this is legal text that we need to follow

**Pull request merged**